<!DOCTYPE html>
<html>
<body>
<h1><center>Sonali Singh</center></h1>
<h2><center>Research Abstracts</center></h2>
<p>ABSTRACT: Understanding how the genome is folded into the highly ordered structure of chromatin that fits into the nucleus while retaining its functionality is an enduring question in biology. The introduction of chromosome conformation capture (3-C) technologies has provided insight into the structural organization of chromatin, but how this correlates to function is still being understood. This review summarizes the current knowledge about chromatin organization as well as recent visualization tools that have enabled further discovery into how chromatin structure regulates function. Topics covered include Hi-C, chromosome territories, TADs, loops, 2D visualization platforms, and 3D genome browsers.</p>
<br>
<p>ABSTRACT: Data from Hi-C techniques have revealed chromatin loops that bring together distant regulatory elements and make up topologically associating domains (TADs). Disrupting enhancer-promoter contacts or modifying the structure of chromatin so that activators and repressors become misplaced has been shown to result in transcriptional misregulation. The loop extrusion hypothesis emphasizes the role of cohesin and transcription factors in loop formation and has proved computationally sound when loops are disturbed. In this paper, we describe SupeRLoop, a supervised reinforcement learning-based algorithm to discover and visualize the policy of loop extrusion. Initial results show the algorithm can predict loops with an accuracy of up to 80% in TF-rich regions and that the CTCF binding motif plays a significant role as a boundary element in the process of loop formation.</p>
</body>
</html>